Realm.xcodeproj/xcshareddata/xcschemes/iOS Dynamic.xcscheme

```diff
@@ -32,13 +32,30 @@
             skipped = "NO">
             <BuildableReference
                BuildableIdentifier = "primary"
-               BlueprintIdentifier = "3F512D8919F9A1F300746016"
-               BuildableName = "iOS 8 Tests.xctest"
-               BlueprintName = "iOS 8 Tests"
+               BlueprintIdentifier = "026B0F041A780680005E26C8"
+               BuildableName = "iOS Dynamic Tests.xctest"
+               BlueprintName = "iOS Dynamic Tests"
                ReferencedContainer = "container:Realm.xcodeproj">
             </BuildableReference>
+            <SkippedTests>
+               <Test
+                  Identifier = "SwiftObjectInterfaceTests/testNSStringFromClassDemangledTopLevelClassNames()">
+               </Test>
+               <Test
+                  Identifier = "SwiftObjectInterfaceTests/testNestedClassNameMangling()">
+               </Test>
+            </SkippedTests>
```

Comment:
I take it all back - these test only work for the 'iOS Tests' target. They will not work for the 'iOS Dynamic' target as they depend on the target name.